data/tabular/uniprot_sentences/meta.yaml

```diff
@@ -35,3 +35,10 @@ templates:
     - |-
       User: Describe the {#function|biological function!} of the {#protein|amino-acid sequence|AA sequence|polypeptide!} {other#}?
       Assistant: {sentences#}.
+    - |-
+      User: What {#protein|amino-acid sequence|AA sequence|polypeptide!} best fits the {#function|biological function!} described in the next sentence(s). {sentences#}
+      Assistant: The {#protein|amino-acid sequence|AA sequence|polypeptide!} that best fits the described function is {other#}.
```

Comment:
```suggestion
      Assistant: The {#amino-acid sequence|AA sequence|polypeptide| protein!} that fits the described function best is {other#}.
```

Reply:
I swapped out the other of the elements, otherwise they are replaced with the same random variable and will get the same value.